.github/workflows/publish-provider-non-family.yml

```
@@ -26,29 +26,40 @@ on:
         default: ubuntu-latest
         required: false
         type: string
-      mirror-to-upbound-registry:
+      mirror:
         description: "If set to true, the xpkg will be mirrored to xpkg.upbound.io"
```

this description is out of date now :) 
```suggestion
        description: "If set to true, the xpkg will be mirrored to the specified mirror-registry"
```